### PR TITLE
Add midstream release notes for triggers

### PR DESCRIPTION
**Pull request by savitaashture**
added release notes for 0.7.0, 0.8.0 and 0.8.1

/cc @nikhil-thomas @khrm @Preeticp

### CODE REVIEW

**Review by vdemeester — COMMENTED** [review]
/lgtm

File: openshift/release/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+## How to upgrade from v0.7.0 :up_arrow:
+```text
+kubectl apply -f https://storage.googleapis.com/tekton-releases/triggers/previous/v0.8.0/release.yaml
+```
+`NOTE: Due to #750, you will have to manually add replicas: 1 to your EL spec before applying the upgrade`
+
+# Tekton Triggers v0.8.1
+
+## Fixes :bug:
+* Handle validation, defaults and working behavior for Replicas ([#751](https://github.com/tektoncd/triggers/pull/751))
```
Comment (khrm) [inline openshift/release/release-notes.md:175]: Not needed

File: openshift/release/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+* Merge annotations before propagation ([#753](https://github.com/tektoncd/triggers/pull/753))
+
+Triggers no longer overwrites annotations set on the underlying deployment and service objects.
+
+* Add a Idle Timeout for EventListener sink ([#755](https://github.com/tektoncd/triggers/pull/755))
+
+EventListeners close idle connections after 120 seconds.
+
+## Docs :book:
+* Add release links to versioned docs for v0.8.0 ([#749](https://github.com/tektoncd/triggers/pull/749))
```
Comment (khrm) [inline openshift/release/release-notes.md:188]: Not needed

File: openshift/release/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+This PR adds the validation and defaults around TriggerCRD.
+
+## Breaking Changes :rotating_light:
+* Switch trigger sa based auth to impersonate ([#705](https://github.com/tektoncd/triggers/pull/705))
+* Switch trigger sa ref from global to namespace scoped ([#704](https://github.com/tektoncd/triggers/pull/704))
+
+The optional EventListenerTrigger based level of authentication for creating Tekton object has had its ServiceAccount reference changed from an ObjectReference to a string ServiceAccountName, effectively enforcing that the ServiceAccount be in the same namespace as the EventListenerTrigger.
+
+## Fixes :bug:
+* Fix Incorrect number of response for Forbidden error ([#673](https://github.com/tektoncd/triggers/pull/673))
```
Comment (khrm) [inline openshift/release/release-notes.md:116]: Not needed.

File: openshift/release/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+* Use Kaniko instead of img in the tutorial to fix permission issue ([#701](https://github.com/tektoncd/triggers/pull/701))
```
Comment (khrm) [inline openshift/release/release-notes.md:117]: Not needed.

File: openshift/release/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+* Cleanup Reconciler Tests in order to pick up Genreconciler updates ([#706](https://github.com/tektoncd/triggers/pull/706))
```
Comment (khrm) [inline openshift/release/release-notes.md:118]: Not needed

File: openshift/release/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+* Fix update deployment when there is a change in replicas ([#715](https://github.com/tektoncd/triggers/pull/715))
+* Add basic syntactical parsing of CEL filters and expressions ([#745](https://github.com/tektoncd/triggers/pull/745))
+
+Perform simple syntax checking of CEL filter and overlays in the Webhook validator, perfunctory syntax validation of the expressions in the interceptor but it won't detect logical errors (expressions that rely on JSON bodies).
+
+## Misc :hammer:
```
Comment (khrm) [inline openshift/release/release-notes.md:124]: Whole section not needed.

File: openshift/release/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
+# Tekton Triggers v0.7.0
+
+## Features
+
+* Add CEL function to parse YAML ([#636](https://github.com/tektoncd/triggers/pull/636))
+
+This PR will add a CEL function named parseYAML that can parse a YAML string into a map of strings to dynamic values
+Syntax: .parseYAML() -> map<string, dyn>
+
+* Expose the incoming request URL to CEL expressions ([#647](https://github.com/tektoncd/triggers/pull/647))
```
Comment (khrm) [inline openshift/release/release-notes.md:10]: Not needed. Internal change.

**Pull request merged**